changelog.d/7209.feature

```diff
@@ -0,0 +1 @@
+[Device Manager] Extend user agent to include device information
```

Comment:
Since the change is SDK side only, I would use `.sdk` extension.

Reply:
Done.